_component_design/buttons.md

```diff
@@ -191,13 +191,87 @@ When submitting forms:
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Button set spacing"] [class:"image bg-light border"] %}
-![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-1.svg){:width="100%"}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-1.svg)
 {% endfigure %}
 {% figure [caption:"Primary, secondary and tertiary button set"] [class:"image bg-light border"] %}
-![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-2.svg){:width="100%"}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-2.svg)
 {% endfigure %}
 {% figure [caption:"Primary and tertiary button set"] [class:"image bg-light border"] %}
-![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-3.svg){:width="100%"}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-3.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Variations
+{: .hxSubSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Drop-down buttons
+
+A drop-down button is preferred when multiple actions are available, with no designated primary action. All actions are  accessed through the drop-down menu.   
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The drop-down button variation"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/drop-buttons-hero.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Specifications
+
+- Drop-down buttons have the same size specifications as simple buttons.
+- The secondary button can be used in conjunction with the primary button or when there are multiple buttons on the same screen.
+- The same sizing rules apply to both secondary and primary buttons.
+- Drop-down buttons and split drop-down buttons feature the same state changes as simple buttons. Refer to [simple button]({{page.url}}#simple-buttons) for specifications.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Drop-down button specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/drop-buttons-specs.svg)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Split drop-down buttons
+
+Split dropdown buttons are preferred when more than one action is associated with a button. The primary action is visible and available on the button itself, while the secondary actions are accessed through the drop-down menu.
```

Comment:
Consider adding a paragraph under this referencing the Actions Menu.

Something like:
See the [Actions Menu](url) for drop-down design specifications.

Reply:
added in lower case form.